tests/coalaCITest.py

```diff
@@ -172,3 +172,19 @@ def test_limit_files_negative(self):
             self.assertFalse(stderr)
             self.assertEqual(retval, 0,
                              'coala must return zero when successful')
+
+    def test_bear_dirs(self):
+        with bear_test_module(), \
```

Comment:
this already adds the test_bears directory, so the test logic below doesnt prove anything

Reply:
Made the changes :+1:  that's for pointing out.

Reply:
but you are still using this context manager ... why ?

Reply:
Shizz, sorry for that. I removed the context manager now the test_bears dir will be added by --bear-dirs flag and we won't be needing the `HiddenTestBear`